solutions/policy-constraint-bundles/std-protectedb-policies/README.md

```diff
@@ -0,0 +1,54 @@
+# Standard Protected B Policy
+
+## Description
+
+Policy bundle for enforcing Protected B compliance.
+
+This bundle is dependant on the `gatekeeper-policies` package.
+
+## Policy Enforcement
+
+By default the policies are set to `dry-run` and will only produce violation notices will not prevent non-compliant resources from being deployed.
+
+For production environments it is recommended the policies be set to `deny`. This can be done by setting the `enforcementAction` field in the packages `Kptfile` to `deny` then running `kpt fn render` on the package to change all policies in the bundle.
+
+This example is what the setting should look like when set to `deny`
+
+```yaml
+apiVersion: kpt.dev/v1
+kind: Kptfile
+...
+pipeline:
+  mutators:
+    - image: gcr.io/kpt-fn/set-enforcement-action:v0.1.0
+      configMap:
+        enforcementAction: deny
+```
+
+### Naming Convention
+All Projects must have the following naming convention `aadmu-string`
+
+```yaml
+client_code: ^(aa|bb|cc)
+env_code: d # corresponds to environment `d` 
+region_code: m # Region Code `m` for montreal
+class_code: (u|a|b)- # unclassified | protected a | protected b
+user_defined_string: '^[a-z][a-z0-9-]*[a-z0-9]$' # user string
+```
+
+### Required Labels
+All Projects must have be labeled with a cost center code that matches the following pattern `^[a-zA-Z]+$`
```

Comment:
All projects must be labeled with a cloud business requirements ID that is represented with 4 digits

Below is the guidance. All cloud service provider environment should be using this standard tag:​

           TagName = ssc_cbrid ​

           Value = 4digit CBR ID

Reply:
this requirement is mandatory to all environements including profile 1, 2, 3, 4, 5, 6